### DISCUSSION TITLE
Can we ignore scrubbing for selective DateTime fields?

### DISCUSSION THREAD

**Discussion by verynewuser** [VerifyTests/Verify#804] [Q&A] [answered]
Verify library srubs date fields by default. Here is the official link. It also provides a way to disable this behavior for a given object, by using `DontScrubDateTimes()` method. Is there a way to disable the scrubbing behavior for only a selected date property in an object, and not for all dates available in that object?

**Comment by SimonCropp** [comment, marked as answer]
nope. this is not supported

**Reply by mattiasnordqvist** [reply]
please support this!

**Reply by SimonCropp** [reply]
happy to consider a pull request
